### ISSUE TITLE
Make initPKSim() function more flexible?

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/OSPSuite-R#983]
At the moment `initPKSim()` (if called without arguments) will only look for the PK-Sim installation **with the same major version** as the version of the ospsuite-R package.
If e.g. the version of the ospsuite-R package is 11 and PK-Sim 10 is installed - `initPKSim()` will fail.

Should we make it more flexible?

At the moment the search strategy is:
1) Look in the registry first, under .`../PK-Sim <ospsuite-R-major Version>` If not found:
2) Look under `Program Files/OSP/PK-Sim <ospsuite-R-major Version>`

If 1) fails - 2) will fail anyway (the only use case when 1) fails but 2) does not would be: somebody installs portable version of PK-Sim under `Program Files/OSP`. The probability of this is extremely low)

So I would suggest to extend the search under 2) like this:
* Loop through all folders `Program Files/OSP/PK-Sim *`
* In every folder: look for **PK-Sim.exe**. If found - return this folder as PK-Sim path

**Comment by msevestre** [comment]
I am not sure that this makes sense. If we have a major version bump, this will definitely mean that the ospsuite.core won't be compatible so this won't work for other reasons.

What about we set the version of ospsuite-r to 11.1 instead? So we are creating an update of ospsuite-r for now and we can continue our life as is

**Comment by Yuri05** [comment]
>What about we set the version of ospsuite-r to 11.1 instead?

Fine for me. @IndrajeetPatil 

Then I would completely remove the part with `.getPathToPKSimInstallDirFromFileSystem`. Cannot imagine we will ever have this situation where `.getPathToPKSimInstallDirFromRegistry` fails, and `.getPathToPKSimInstallDirFromFileSystem` succeeds after that.
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/88ee241f4585a17f55c85e28a81aa0eda952db04/R/utilities-pksim.R#L162-L175

**Comment by PavelBal** [comment]
Yes, I just unpack it to the default folder where all other versions are:

<img width="140" alt="image" src="https://user-images.githubusercontent.com/25903040/170052632-6e8eb50b-8f30-45b6-8fb9-49149fb715c9.png">

One of the reasons was exactly that the R package could always find the correct version.

**Comment by Yuri05** [comment]
ok... well, let's keep it AS IS then. Does not hurt.
